Modules/Messages/src/Messages.Domain/Entities/Relationship.cs

```diff
@@ -24,6 +25,15 @@ private Relationship()
     public DateTime CreatedAt { get; }
 
     public RelationshipStatus Status { get; }
+
+    public void EnsureSendingMessagesIsAllowed(int numberOfUnreceivedMessagesFromActiveIdentity, int maxNumberOfUnreceivedMessagesFromOneSender)
```

Comment:
Tests for this method are missing

Reply:
Should I do a simple domain test for the method in isolation, or should I test the entire handler?

Reply:
Changes made.